state-chain/chains/src/mocks.rs

```diff
@@ -196,9 +196,9 @@ pub struct MockTransaction;
 impl FeeRefundCalculator<MockEthereum> for MockTransaction {
 	fn return_fee_refund(
 		&self,
-		_fee_paid: <MockEthereum as Chain>::TransactionFee,
+		fee_paid: <MockEthereum as Chain>::TransactionFee,
 	) -> <MockEthereum as Chain>::ChainAmount {
-		<MockEthereum as Chain>::ChainAmount::default()
+		fee_paid.effective_gas_price * fee_paid.gas_used
```

Comment:
The test that was testing refunds before was actually just testing default = default 😬 ... fortunately, it does work.